Gemfile

```diff
@@ -29,7 +29,8 @@ gem 'sidetiq'
 gem 'sidekiq-unique-jobs'
 gem 'sidekiq-failures'
 gem 'sinatra'
-gem 'aws-sdk', '~> 2'
+gem 'aws-sdk-v1'
```

Comment:
Ah I had some trouble trying to get pre-signed url to pass to component with V2, I'll mess around a bit more and use v2